tests/api/test_download.py

```diff
@@ -69,10 +69,45 @@ def test_download_image_with_session(self):
     def test_download_all_video_frames(self):
         with tempfile.NamedTemporaryFile(suffix='.avi') as file:
             original = _generate_video(file.name)
-            frames = list(download.download_all_video_frames(file.name, as_pil_image=False))
+            frames = list(download.download_all_video_frames(file.name))
             for frame, orig in zip(frames, original):
                 assert _images_equal(frame, orig)
 
+    def test_download_video_frame(self):
+        with tempfile.NamedTemporaryFile(suffix='.avi') as file:
+            original = _generate_video(file.name)
+            for timestamp, orig in enumerate(original):
+                frame = download.download_video_frame(file.name, timestamp)
+                assert _images_equal(frame, orig)
+
+    def test_download_video_frame_fps(self):
+        for fps in range(1, 5):
+            with tempfile.NamedTemporaryFile(suffix='.avi') as file:
```

Comment:
Add a subtest here:
```python
with self.subTest(fps=fps):
```
It allows you to see directly in the unittesting framework which of the subtest failed (e.g. all 5 or just one of them and which).
Documentation: https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest

The same applies also to the other unittests.